src/app/store/reducers/sample.reducers.js

```diff
@@ -0,0 +1,21 @@
+import * as Actions from '../actions/sample.actions';
+
+const initialState = {
+  hasInitialState: true,
+};
+
+const sample = (state = initialState, action) => {
+  switch (action.type) {
+    case Actions.SAMPLE_ACTION:
+      return { nothing: null };
```

Comment:
As much as possible I'd advise you use variables that people can relate to.

If I wasn't someone that knew Redux, this would be confusing and you see this part right here is what makes redux redux... because how you construct your state informs how well your app works